src/binarylane/console/util.py

```diff
@@ -0,0 +1,24 @@
+from __future__ import annotations
+
+from binarylane.pycompat.importlib import metadata
+
+from binarylane.client import AuthenticatedClient
+from binarylane.config import Config
+
+
+def get_version() -> str:
+    try:
+        return metadata.distribution("binarylane-cli").version
+    except metadata.PackageNotFoundError:
+        return "dev"
+
+
+def create_client(config: Config) -> AuthenticatedClient:
+    return AuthenticatedClient(
+        base_url=config.api_url,
+        token=config.api_token,
+        verify_ssl=not config.api_development,
+        timeout=5.0,
+        raise_on_unexpected_status=False,
+        headers={"User-Agent": f"binarylane-cli/{get_version()}"},
```

Comment:
it would be handy to get the platform and python version here too, something that is generic enough that it can't identify the user, maybe `sys.platform` and `sys.version` in brackets  after `binarylane-cli/{get_version()} ({sys.platform}/{sys.version()})` which would give something like

```
binarylane-cli/0.13.0 (win32/3.10.4 (tags/v3.10.4:9d38120, Mar 23 2022, 23:13:41) [MSC v.1929 64 bit (AMD64)])
```

Reply:
Bit long? feels like the initial `win32/3.10.4` would do